doc/src/manual/performance-tips.md

```diff
@@ -249,11 +268,11 @@ julia> typeof(m.a)
 Float64
 ```
 
-The fact that the type of `m.a` is known from `m`'s type--coupled with the fact that its type
-cannot change mid-function--allows the compiler to generate highly-optimized code for objects
+The fact that the type of `m.a` is known from `m`'s type—coupled with the fact that its type
```

Comment:
While you're at it, add spaces around the dashes?

Reply:
It varies.   e.g. [this source](http://www.thepunctuationguide.com/em-dash.html) says that in AP style you have a space before and after em dashes, whereas in [Chicago style you have no spaces](https://apvschicago.com/2011/05/em-dashes-and-ellipses-closed-or-spaced.html).

(In a monospaced font, I find the em dash much more readable if it is set off with spaces, because the monospaced em is too narrow.  The manual is displayed in a proportional font, of course, but we edit it in monospace.)

Either way seems fine to me, as long as we are consistent.